### PR TITLE
fixing some test error and removing array credentials from config file

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

File: tests/config.ini [hunk 1]
```diff
@@ -2,16 +2,14 @@
 
 # please provide the Nimble Array credentials against which the test case will be run. Note that the password is plain text and hence when the same is released to public github it will be removed
 [NimbleArrayCredentials] #section name
-hostname: 10.18.161.229
```
Comment (ar-india) [inline tests/config.ini:5]: removing array credentials

File: tests/nimbleclientbase.py [hunk 2]
```diff
@@ -22,7 +22,6 @@
 log_folder = os.path.abspath(os.path.dirname(__file__)) + "//" + "logs".strip()
 
 try:
-    print("hello")
```
Comment (ar-india) [inline tests/nimbleclientbase.py:25]: unwanted print removed

File: tests/test_initiator_and_initiatorgroups.py [hunk 3]
```diff
@@ -199,13 +199,6 @@ def test_get_initiatorgroups(setup_teardown_for_each_test):
         igrps = nimosclientbase.get_nimos_client(
         ).initiator_groups.list(detail=True, pageSize=2)
 
-        # igrps_get = nimosclientbase.get_nimos_client().initiator_groups.get(id=None,
```
Comment (ar-india) [inline tests/test_initiator_and_initiatorgroups.py:202]: removed this commented code

**Review by suneeth51 — APPROVED** [review]
lgtm..

**Pull request merged**